src/app/js/lib/components/FieldDisplayInResourceInput.js

```diff
@@ -1,20 +1,10 @@
 import React from 'react';
-import translate from 'redux-polyglot/translate';
-import { Field } from 'redux-form';
 
 import FormCheckboxField from './FormCheckboxField';
-import { polyglot as polyglotPropTypes } from '../../propTypes';
+import FieldInput from './FieldInput';
 
-export const FieldDisplayInResourceInputComponent = ({ p: polyglot }) => (
-    <Field
-        name="display_in_resource"
-        component={FormCheckboxField}
-        label={polyglot.t('field_display_in_resource')}
-    />
-);
-
-FieldDisplayInResourceInputComponent.propTypes = {
-    p: polyglotPropTypes.isRequired,
-};
-
-export default translate(FieldDisplayInResourceInputComponent);
+export default () => <FieldInput
+    name="display_in_resource"
```

Comment:
the only difference with the other component are int the props, do not create a component just for that